Config/BuildSettings.swift

```diff
@@ -55,6 +55,7 @@ final class BuildSettings: NSObject {
         "app.element.io": [],
         "staging.element.io": [],
         "develop.element.io": [],
+        "mobile.element.io": ["/config/config"],
```

Comment:
The config/config path has disappeared according to the element-web PR.
It should be something like `[]` or `[""]`

Reply:
Uh yes, you're right. I was badly looking at https://github.com/vector-im/element-web/issues/14751